lambdatimes/src/components/Content/Content.js

```diff
@@ -19,14 +19,16 @@ export default class Content extends Component {
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
    this.setState({
-    tabs: tabData,
-    cards: cardData,
+    tabs: [...tabData],
+    cards: [...cardData],
```

Comment:
another good use of spread operator here, could you explain the reasoning for using a spread operator and what advantage it gives you